examples/basic/loc.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: delivery.ocm.software/v1alpha1
+kind: Localization
+metadata:
+  name: deployment
+spec:
+  interval: 10m0s
+  sourceRef:
+    kind: ComponentVersion
```

Comment:
I really like this. Like, it's more apparent what object we are requiring data from instead of knowing that we produce a snapshot which you will need which makes less sense.

This is good stuff! I like this. :)